### PR TITLE
Client refactoring

### LINKED ISSUE #12: Refactor blocks world client

### LINKED ISSUE #13: Enforce single click constraint.
We should add a "Issue Command" button to transmit the "verbal" command and the pointed-to location in one discrete packet.

### PR DESCRIPTION
**Pull request by Lewis-Kelley**
Close #12 
Close #13 
Some progress on #11

### CODE REVIEW

**Comment by kgroble** [comment]
part regarding #13 looks good, still going through the other changes

File: docs/README.org [hunk 1]
```diff
@@ -0,0 +1,10 @@
+* Documentation
+Note that many of these images are generated using PlantUML for the
+=*.puml= files.
+** Sequence Diagrams:
+*** Connecting
+[[./connecting.png]]
+*** Gameplay
+[[./gameplay.png]]
```
Comment (kgroble) [inline docs/README.org]: Did you forget to commit `gameplay.png`?

**Comment by jcolemang** [comment]
For future reference: One of the merge conflicts in `game.html` is apparently in part caused by a script tag sitting in the middle of the page. It should stay there unless the JavaScript above the tag is factored into a different file, because `game.js` depends on the initialization done in `game.html`.

File: .travis.yml [hunk 2]
```diff
@@ -4,7 +4,9 @@ python:
   - "3.6"
 
 install:
-  - sudo pip install pytest python-socketio
+  - sudo pip install pytest python-socketio pylint psycopg2 tornado eventlet
```
Comment (jcolemang) [inline .travis.yml]: Would you put these into a `requirements.txt` file?

**Review by jcolemang — APPROVED** [review]

**Review by kgroble — APPROVED** [review]

**Pull request merged**